### PR TITLE
[argparse] Bumps argparse to version v2.2.

### PR DESCRIPTION
**Pull request by marcovc**
Bumps argparse to version v2.2 (latest).

### CODE REVIEW

**Review by PhoebeHui — APPROVED** [review]
Thanks for your updates!

File: ports/argparse/portfile.cmake [hunk 1]
```diff
@@ -3,21 +3,20 @@
...
-    REF 9903a22904fed8176c4a1f69c4b691304b23c78e #2021-03-04
-    SHA512 da50761471341d558ed0c336bf78f80a704ec25c15ace23fb3a92d81ddbf2e4c2fd4f6c4226f088382f41418d58d5ef9b57cdd6e9689be5d83dba6adb2ac0aba
+    REF b0cb28ab177cdbd19d439c02cdc217bdfc508ba9 #2021-09-14
+    SHA512 247f27aa45994ee4629b947e3fe1b1df8fae4e773992c740a7c3d544e45ce83a68ecdcfdedf38c461ccb81cf93d88dfee21dc4f7d027677a3c7d930170446971
     HEAD_REF master
 )
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
```
Comment (PhoebeHui) [inline ports/argparse/portfile.cmake:12]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"

```

File: ports/argparse/portfile.cmake [hunk 2]
```diff
@@ -3,21 +3,20 @@
...
         -DARGPARSE_BUILD_TESTS=OFF
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT})
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/${PORT})
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug ${CURRENT_PACKAGES_DIR}/lib)
```
Comment (PhoebeHui) [inline ports/argparse/portfile.cmake:21]:
```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug" "${CURRENT_PACKAGES_DIR}/lib")

```

**Comment by PhoebeHui** [comment]
@marcovc, could you address the review suggestions, submit the changes, and update the version via "./vcpkg x-add-version --overwrite-version argparse' and submit the changes again?

**Review by PhoebeHui — APPROVED** [review]
Thanks for your updates!

**Comment by marcovc** [comment]
Thank you. How to merge this PR (I haven't write access)?

File: ports/argparse/portfile.cmake [hunk 3]
```diff
@@ -3,23 +3,22 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO p-ranav/argparse
-    REF 9903a22904fed8176c4a1f69c4b691304b23c78e #2021-03-04
-    SHA512 da50761471341d558ed0c336bf78f80a704ec25c15ace23fb3a92d81ddbf2e4c2fd4f6c4226f088382f41418d58d5ef9b57cdd6e9689be5d83dba6adb2ac0aba
+    REF b0cb28ab177cdbd19d439c02cdc217bdfc508ba9 #2021-09-14
```
Comment (strega-nil-ms) [inline ports/argparse/portfile.cmake:6]: This should probably be:

```suggestion
    REF b0cb28ab177cdbd19d439c02cdc217bdfc508ba9 # v2.2
```

File: ports/argparse/vcpkg.json [hunk 4]
```diff
@@ -1,7 +1,17 @@
 {
   "name": "argparse",
-  "version-date": "2021-03-04",
+  "version-date": "2021-09-14",
```
Comment (strega-nil-ms) [inline ports/argparse/vcpkg.json:3]:
```suggestion
  "version": "2.2",
```

given that they release versions, we should use those versions

**Review by strega-nil-ms — CHANGES_REQUESTED** [review]

**Comment by BillyONeal** [comment]
Thanks for the version update!

**Pull request merged**